### PR TITLE
Add a folder with the printf function and its functionalities

### PR DESCRIPTION
**Pull request by ignaciocastro347**

### CODE REVIEW

File: printf/_printf.c [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#include "_printf.h"
+#include <stdarg.h>
```
Comment (josealvarezzok) [inline printf/_printf.c:2]: Qnyr obova cbar yn yvoerevn ra ry .u :Q

File: printf/_printf.h [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+ *@c: The format specification
+ *@f: The function associated
+ */
+typedef struct print_function
+{
+	char c;
+	int (*f)(va_list);
+} pfunc_t;
+
+/*############### MAP FUNCTION #################*/
```
Comment (josealvarezzok) [inline printf/_printf.h:21]:
```suggestion
/* ############## MAP FUNCTION ################# */
```

File: printf/_putchar.c [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#include "_printf.h"
+#include <unistd.h>
```
Comment (josealvarezzok) [inline printf/_putchar.c:2]: Same here

File: printf/main.c [hunk 4]
```diff
@@ -0,0 +1,17 @@
+#include "main.h"
```
Comment (josealvarezzok) [inline printf/main.c:1]: Change name to `test_printf.c`

File: shell.h [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#ifndef SIMPLE_SHELL
+#define SIMPLE_SHELL
+#define  _GNU_SOURCE
+
+#include <stdio.h>
+#include <string.h>
```
Comment (BesterGG) [inline shell.h:6]: delete `<string.h>`

**Review by josealvarezzok — APPROVED** [review]
Well done!

**Pull request merged**